ui-manchette-with-spacetimechart/src/assets/sampleData.ts

```diff
@@ -188,3 +189,24 @@ export const SAMPLE_PATHS_DATA: ProjectPathTrainResult[] = [
     ],
   },
 ];
+
+export const SAMPLE_WORK_SCHEDULES: WorkSchedule[] = [
+  {
+    type: 'TRACK',
+    timeStart: '2024-06-26T14:00:00Z',
+    timeEnd: '2024-06-26T14:15:00Z',
+    intersections: [
```

Comment:
Nit: maybe `spaceRanges` would be a better name?

I'm a bit puzzled by "intersections", not sure what intersects what and I don't think the component should care which mathematical operation was performed to obtain this data.